src/model/LabyrinthMatrix.java

```diff
@@ -1,91 +0,0 @@
-package model;
-
-import java.io.File;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Scanner;
-
-public class LabyrinthMatrix extends LabyrinthImpl {
-
-	private int matrix[][];
-
-	public LabyrinthMatrix(String path) {
-		Scanner sc;
-		try {
```

Comment:
Have a look on ARM: try-with-resources https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html (Java 7 feature) may save you for writing boilerplate code.